src/tiny_slam/tiny_grid_cells.h

```diff
@@ -1,24 +1,61 @@
+/*!
+ * \file
+ * \brief Definition of classes file: BaseTinyCell, AvgTinyCell (are inherited from GidCell), TinyBaseCellFactory, TinyAvgCellFactory (from GridCellFactory)
+ *
+ * Two classes from this file (inherited from GridCell) presents two different ways to calculate a probability of cell occupation.
```

Comment:
"The classes inserted form <...> implements different approaches of grid cell's occupancy maintaining"